### ISSUE TITLE
[Action to craft a Project Proposal] User Stories of CNCF Projects integrating Environmental Sustainability

### ISSUE THREAD

**Issue by leonardpahlke** [cncf/tag-env-sustainability#328]
<sup>This is a tracking issue for the action item proposed and accepted for the TAG ENV 2024 roadmap see https://github.com/cncf/tag-env-sustainability/issues/321. Proposed originally [here](https://github.com/cncf/tag-env-sustainability/discussions/277#discussioncomment-7635965).</sup>

**Accepted project proposal summary**: Write blogs, documentation or a whitepaper. Documenting user stories of CNCF projects transforming and enhancing their projects to have environmental sustainability in mind.

**Next steps**:
- [ ] Project proposal https://github.com/cncf/tag-env-sustainability/issues/new?assignees=&labels=triage-required&projects=&template=project-suggestion.md&title=%5BPROEJCT+PROPOSAL+%2F+SUGGESTION%5D+some+descriptive+title - used to discuss the project, find additional collaborators and have a tracking issue for the project.
- [ ] Have one or more leads for the project to report about the progress every other week.
- [ ] Details added to the [TAG ENV website (new page next to working groups)](https://tag-env-sustainability.cncf.io/about/)
- [ ] Project details announced

@nikimanoledaki  - as author, it would be great if you could follow the steps described above to start the project. If not, please comment below, in which case the project will likely be canceled. If you need support, comment below or get in touch via Slack. Thank you very much!

@vallss - you mentioned interest helping on this one too, perhaps you could take the lead / share the lead too. CC a few others that may be interested too @ams0 @akelity @AndreaGiardini 

--

cc @mkorbi @catblade 
cc WG Comms @caradelia @patricia-cahill

**Comment by leonardpahlke** [comment]
Great! The issue can be part of this. Reads to me as a good first action :)

**Comment by akelity** [comment]
Hi @leonardpahlke, it's a busy time, but I'm more than happy to contribute (even though, as you can see, I might be particularly slow in responding). 

Next week, I'll gladly take a look and figure out how I can help! :)

**Comment by leonardpahlke** [comment]
We just added two new GitHub issue templates to [propose](https://github.com/cncf/tag-env-sustainability/issues/new?assignees=&labels=triage-required&projects=&template=project-suggestion.md&title=%5BPROEJCT+PROPOSAL+%2F+SUGGESTION%5D+some+descriptive+title) and [track](https://github.com/cncf/tag-env-sustainability/issues/new?assignees=&labels=tracking&projects=&template=project-tracking.md&title=%5BPROJECT+TRACKING%5D+some+descriptive+title) projects. The proposal topic is used to define and discuss the project details. The tracking output is used to track the ongoing progress until the project is completed. I linked the proposal issue in the todo items of this issue. After this issue is completed, we can open the tracking issue and start working on the project. I hope this helps.

**Comment by werstatyw** [comment]
Hi guys, is this applies to already existing projects or not? I'm currently developing a k8s startup which will include sustainability option.

**Comment by bertysentry** [comment]
Hey! Sentry Software here! We can tell the story of how we leveraged OpenTelemetry (and semantic conventions) to optimize the temperature of our data center and reduce its carbon emissions, in pure software. It's a story we already presented at GrafanaCon a year ago, but I think it's still relevant 😊

Edit: Actually, there is already [a paper published on this story at Grafana](https://grafana.com/blog/2023/10/02/reducing-data-center-carbon-emissions-with-hardware-sentry-grafana-and-opentelemetry/). You guys tell me if you still want this story to be included here, we will provide the content:!
